web/declarations.d.ts

```diff
@@ -0,0 +1 @@
+declare let __APP_VERSION__: string;
```

Comment:
We have a 
```
declare global {
    interface Window {
  ..
...
...
}
```
in our src/config/index.ts file. Not sure it makes sense to put this there too. If it doesn't, perhaps we can move the global declaration in to this new file so that we keep things tidy.
What do you think?

Reply:
You're right, I may have overlooked that global class exist here. https://github.com/reearth/reearth/pull/428/commits/ff919e265894de21e1cab39955b16e487eddce9e should fix that.